lib/account/pages/write_review_page.dart

```diff
@@ -148,32 +149,50 @@ class _WriteReviewPageState extends State<WriteReviewPage> {
   }
 
   void onRatingUpdate(double rating) {
+    ratingChangedManually = true;
     setState(() {
       _review!.rating = rating.toInt();
     });
   }
 
+  void _updateOverallReview() {
+    if (!ratingChangedManually) {
+      final Iterable<int?> givenCategoryRatings = <int?>[
+        _review!.comfortRating,
+        _review!.safetyRating,
+        _review!.hospitalityRating,
+        _review!.reliabilityRating
+      ].where((int? element) => element != null);
+      _review!.rating =
+          givenCategoryRatings.reduce((int? value, int? element) => value! + element!)! ~/ givenCategoryRatings.length;
```

Comment:
this is always rounding down, is this what we want, i think it' confusing if i select 3 stars everywhere except one with 2 and the overall rating is 2

Reply:
Why are we rounding at all?

Reply:
Because the user can't input double values, it'd be weird if they were only achievable this way?

Reply:
I think it would be fun. The average might be a whole half star away from what we will show. That's not particularly accurate...

Reply:
yes that's true but since the user can't input double values it makes more sense this way in my opinion.